src/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php

```diff
@@ -1203,9 +1203,7 @@ public function get_collection_params() {
 	 */
 	protected function check_is_taxonomy_allowed( $taxonomy ) {
 		$taxonomy_obj = get_taxonomy( $taxonomy );
-		if ( $taxonomy_obj && ! empty( $taxonomy_obj->show_in_rest ) ) {
-			return true;
-		}
-		return false;
+
+		return $taxonomy_obj && ! empty( $taxonomy_obj->show_in_rest );
```

Comment:
While this is absolutely correct, the old one was easier to understand in my eyes. Not absolutely convinced here, but not against it either.

Maybe it just feels wrong when seeing them side by side.